test_data/yast/firstboot/yast2_firstboot.yaml

```diff
@@ -6,3 +6,4 @@ clients:
   - firstboot_user
   - firstboot_root
   - firstboot_finish
+default_language: "English (US)"
```

Comment:
Shall we separate default language for the keyboard and language for the system/installer?